src/univariate/discrete/discreteuniform.jl

```diff
@@ -73,7 +87,7 @@ modes(d::DiscreteUniform) = [d.a:d.b]
 pdf(d::DiscreteUniform, x::Real) = insupport(d, x) ? d.pv : zero(d.pv)
 logpdf(d::DiscreteUniform, x::Real) = log(pdf(d, x))
 
-function cdf(d::DiscreteUniform, x::Int)
+function cdf(d::DiscreteUniform, x::Integer)
```

Comment:
Unrelated?
```suggestion
function cdf(d::DiscreteUniform, x::Int)
```

Reply:
Will adapt this when reverting the DiscreteUniform to non-parametric.